packages/orchestrator/lib/clients/types.ts

```diff
@@ -76,6 +77,26 @@ interface TaskCommon extends TaskCommonFields {
     isAction(this: OrchestratorTask): this is TaskAction;
     isOnEvent(this: OrchestratorTask): this is TaskOnEvent;
     isSyncAbort(this: OrchestratorTask): this is TaskSyncAbort;
+    isAbort(this: OrchestratorTask): this is TaskAbort;
+}
+export interface TaskAbort extends TaskCommon, AbortArgs {}
```

Comment:
I could have gone with a abort type per type of script (ie: TaskSyncAbort, TaskActionAbort, TaskWebhookAbort, etc...) or a single type with optional field to deal with TaskSyncAbort which is a bit different. 
I decided to go for a middle ground solution with TaskSyncAbort and a generic TaskAbort for everything else